packages/building-rollup/README.md

```diff
@@ -187,16 +187,26 @@ If you need to customize the output of your `index.html` you can pass extra opti
 
 ```javascript
 import { createDefaultConfig } from '@open-wc/building-rollup';
+import { indexHTMLPlugin } from '@open-wc/rollup-plugin-index-html';
```

Comment:
We need to update the sentence at line 186 as well, since we don't recommend passing custom options anymore.